iep-module-eureka/README.md

```diff
@@ -9,11 +9,15 @@ Guice module to configure [eureka](https://github.com/Netflix/eureka). It will d
 
 ## Gradle
 
+The `iep-module-archaius1` dependency sets up the bridge so that Eureka, which is built on
+archaius1, can access the properties necessary to register and begin heartbeat.
```

Comment:
This isn't always done. The reason we do not do it automatically as a transitive dependency is that the app may be configuring archaius1 in some other way.